src/app/components/common/StatusBadge.tsx

```diff
@@ -58,6 +58,9 @@ const StyledBadge = styled(Box, {
     padding: 4,
     paddingLeft: 10,
     paddingRight: 5,
+    [theme.breakpoints.down('sm')]: {
+      padding: 4,
+    },
```

Comment:
I'll fix: spacing here is not related to mobile. It should be displayed depending on hideLabel